tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -29,7 +29,7 @@ Feature: full text search
       | spaces           |
 
 
-  Scenario Outline: sharee searches a file tagged by sharer using tag
+  Scenario Outline: sharee searches files using a tag
```

Comment:
In this scenario sharee tags another file from a `share-with-me resource` with the same tag name as sharer Alice tagged her shared file 
if we change the scenario description then it will be better.

Reply:
ok